templates/circle-agent/src/lib/WalletService.ts

```
@@ -0,0 +1,208 @@
+        id: tx.id,
+        state: tx.state,
+        txHash: tx.txHash,
+        createDate: tx.createDate,
+        updateDate: tx.updateDate
+      };
+    } catch (error) {
+      console.error('Error checking transaction status:', error);
+      throw error;
+    }
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)